### DISCUSSION TITLE
Starting dedicaded standalone server without graphics ?

### DISCUSSION THREAD

**Discussion by n0bleman** [Planimeter/grid-sdk#115] [Q&A] [answered]
How i can start dedicated standalone server witout graphics ? And where player parameters of server player speed hp and more
map command start with client i want start server on another hosting

**Comment by andrewmcwatters** [comment, marked as answer]
Great question, @n0bleman. You can run Grid with `love . --dedicated` which will run headless, without graphics.

The default player class uses the `moveSpeed` network variable, which can be changed.

**Reply by n0bleman** [reply]
Im Using windows 7 i use run.batto start engine
C:\Program Files\LOVE\love.exe" --console "%cd%"exit
i try 
"C:\Program Files\LOVE\love.exe" --dedicated "%cd%"
exit
dont work
1.All Collision,HP and movements on serverside ??? Its impossible to hack?
2.how to separate the server side and the client side and run each separately as in cs strike game

**Reply by andrewmcwatters** [reply]
Hi @n0bleman, I believe you need your `%cd%` variable before the arguments. I would try that.

1. Yes, network variables are server authoritative. It is not possible to cheat this way.
2. You would run two instances. Your dedicated server as one instance, and a client seperately which connects to the first.
